### PR TITLE
Create ShadowCopyDeletion.yml

### PR DESCRIPTION
**Pull request by Bert-JanP**
Required items, please complete
   
   Change(s):
   - Added new Hunting query

   Reason for Change(s):
   - Addition to the current query pack

   Version Updated:
   - Yes

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - No

### CODE REVIEW

**Comment by v-spadarthi** [comment]
@aprakash13 : Please Review the Hunting queries and provide your comments. Thanks!!

**Comment by NikTripathi** [comment]
@aprakash13 please have a look.

File: Hunting Queries/Microsoft 365 Defender/Ransomware/ShadowCopyDeletion.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  @'reg add "HKLM\SOFTWARE\Policies\Microsoft\Windows NT\SystemRestore" /v "DisableSR" /t "REG_DWORD" /d "1" /f', 
+  @'reg add "HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\SystemRestore" /v "DisableConfig" /t "REG_DWORD" /d "1" /f', 
+  @'reg add "HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\SystemRestore" /v "DisableSR" /t "REG_DWORD" /d "1" /f', 
+  @'reg delete "HKLM\SOFTWARE\Policies\Microsoft\Windows NT\SystemRestore" /v "DisableConfig" /f >nul 2>&1', 
+  @'reg delete "HKLM\SOFTWARE\Policies\Microsoft\Windows NT\SystemRestore" /v "DisableSR" /f >nul 2>&1', 
+  @'reg delete "HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\SystemRestore" /v "DisableConfig" /f >nul 2>&1', 
+  @'reg delete "HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\SystemRestore" /v "DisableSR" /f >nul 2>&1']);
+  DeviceProcessEvents
+  | where ProcessCommandLine has_any (CommonRansomwareExecutionCommands)
+  | project-reorder Timestamp, ProcessCommandLine, DeviceName, AccountName
```
Comment (shainw) [inline Hunting Queries/Microsoft 365 Defender/Ransomware/ShadowCopyDeletion.yml:34]: @Bert-JanP  -  Please change Timestamp to standard time column of TimeGenerated.  They are equal, but TimeGenerated is the standard.

**Review by shainw — CHANGES_REQUESTED** [review]

File: Hunting Queries/Microsoft 365 Defender/Ransomware/ShadowCopyDeletion.yml (hunk 1 above)
Comment (shainw) [reply Hunting Queries/Microsoft 365 Defender/Ransomware/ShadowCopyDeletion.yml:34]: @Bert-JanP - Nevermind, missed this was in M365 Hunting folder, Timestamp is good here.

**Review by shainw — APPROVED** [review]

**Pull request merged**